android/app/build.gradle

```diff
@@ -208,14 +208,14 @@ android {
         dev {
             minSdkVersion rootProject.ext.minSdkVersion
             targetSdkVersion rootProject.ext.targetSdkVersion
-            resValue "string", "app_name", "BetterSocial Dev"
+            resValue "string", "app_name", "HelioDev"
             applicationIdSuffix ".dev"
             resValue "string", "build_config_package", "org.bettersocial"
         }
         prod {
             minSdkVersion rootProject.ext.minSdkVersion
             targetSdkVersion rootProject.ext.targetSdkVersion
-            resValue "string", "app_name", "BetterSocial"
+            resValue "string", "app_name", "Helio"
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The application name for the production version has been changed from "BetterSocial" to "Helio". This change seems fine as long as all stakeholders are aware and approve of the new name.

```diff
- resValue "string", "app_name", "BetterSocial"
+ resValue "string", "app_name", "Helio"
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->